### ISSUE TITLE
Module's function gets rewritten

### ISSUE THREAD

**Issue by fildahui** [rollup/rollup#3382]
- Rollup Version: 1.31.0
- Operating System (or Browser): Windows 10
- Node Version (if applicable): 10.13.0
- Link to reproduction (IMPORTANT, read below): https://repl.it/@fildahui/FantasticLongPoint

I'm creating a react component using `fabric-ui-react` lib and for some reason a function from fabric modules gets its body rewritten when bundling (and won't work after that). I'm using two plugins `rollup-plugin-babel` and  `@rollup/plugin-node-resolve`.
Babel plugin has the option `exclude: 'node_modules/**'` so it's not affecting the dependencies.

This is the function of fabric module:
![image](https://user-images.githubusercontent.com/17295499/74224610-18f73d00-4cb1-11ea-81e9-82df491f9c36.png)

This is the function _after_ bundling with rollup:
![image](https://user-images.githubusercontent.com/17295499/74224671-2dd3d080-4cb1-11ea-906c-506a01d7c701.png)

### Expected Behavior

Modules functions shouldn't be changed

### Actual Behavior
 
The function body is rewritten and many lines are stripped out

I noticed that if I manually edit the source module and change this 
```javascript
var _a = getIcon(iconName) || {
        subset: {},
        code: undefined
}, code = _a.code, subset = _a.subset;
```
to this:
```javascript
var _a = getIcon(iconName);
_a = _a || {
        subset: {},
        code: undefined
};
var code = _a.code, subset = _a.subset;
```

The bundled output is correct and the function is not changed.

**Comment by fildahui** [comment]
Hi @lukastaegert is there any update on this?

**Comment by lukastaegert** [comment]
Did not find time to look into this yet

**Comment by lukastaegert** [comment]
Very simplified repro: https://rollupjs.org/repl/?version=2.0.4&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmZ1bmN0aW9uJTIwZ2V0SWNvbihuYW1lKSUyMCU3QiU1Q24lMjAlMjAlMjAlMjB2YXIlMjBpY29uJTIwJTNEJTIwdW5kZWZpbmVkJTNCJTVDbiU1Q3QlMjAlMjBpY29uJTIwJTNEJTIwJTdCY29kZSUzQSUyMHRydWUlN0QlM0IlNUNuJTIwJTIwJTIwJTIwcmV0dXJuJTIwaWNvbiUzQiU1Q24lN0QlNUNuJTVDbmV4cG9ydCUyMGZ1bmN0aW9uJTIwbWFpbihpY29uTmFtZSklMjAlN0IlNUNuJTIwJTIwJTIwJTIwdmFyJTIwX2ElMjAlM0QlMjBnZXRJY29uKGljb25OYW1lKSUyMCU3QyU3QyUyMCU3QiU1Q24lMjAlMjAlMjAlMjAlMjAlMjAlMjAlMjBjb2RlJTNBJTIwdW5kZWZpbmVkJTVDbiUyMCUyMCUyMCUyMCU3RCUzQiU1Q24lMjAlMjAlMjAlMjBpZiUyMCghX2EuY29kZSklMjAlN0IlNUNuJTIwJTIwJTIwJTIwJTIwJTIwJTIwJTIwcmV0dXJuJTIwJ2Jyb2tlbiclM0IlNUNuJTIwJTIwJTIwJTIwJTdEJTVDbiUyMCUyMCUyMCUyMHJldHVybiUyMCd3b3JrcyclM0IlNUNuJTdEJTNCJTVDbiUyMiUyQyUyMmlzRW50cnklMjIlM0F0cnVlJTdEJTVEJTJDJTIyb3B0aW9ucyUyMiUzQSU3QiUyMmZvcm1hdCUyMiUzQSUyMmVzbSUyMiUyQyUyMm5hbWUlMjIlM0ElMjJteUJ1bmRsZSUyMiUyQyUyMmFtZCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyJTIyJTdEJTJDJTIyZ2xvYmFscyUyMiUzQSU3QiU3RCU3RCUyQyUyMmV4YW1wbGUlMjIlM0FudWxsJTdE

**Comment by lukastaegert** [comment]
Thanks for spotting it, there was indeed a small but nasty bug at the source of it, fix at #3438
